magazine.html

```diff
@@ -0,0 +1,203 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+        
+        <style>
+            body {
+                padding-top: 50px;
+                padding-bottom: 20px;
+            }
+            .card{
+                margin-top: 50px;
+                text-align: center;
+            }
+            
+            
+            
+          
+            
+            
+        </style>
+		<link rel="stylesheet" href="css/theme-bootstrap.css">
+
+        <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="index.html"><span class="sr-only">(current)</span>MegaPoem</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+		  <ul class="nav navbar-nav">
+			<li class="dropdown">
+			  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Poems <span class="caret"></span></a>
+			  <ul class="dropdown-menu">
+				 
+				 
+				<li><a href="poems.html">Title</a></li>
+				<li><a href="poems-subject.html">Subject Matter</a></li>
+				<li><a href="#">Popularity</a></li>
+				
+				<li><a href="#">Location</a></li>
+			  </ul>
+			</li>
+			<li><a href="magazine.html">Magazine</a></li>
+			<li><a href="about.html">About</a></li>  
+			
+		  </ul>	
+          <ul class="nav navbar-nav navbar-right">
+        		<li><a href="#"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Sign In</a></li>
+      	  </ul>  
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <div class="card">
+            <div class="card-block">
+                <h4 class="card-title">MegaPoem ISH I.</h4>    
+            </div>
+            <img class="card-img" src="../img/PoemCoverMonet2.jpg" alt="Card image" width="150px" height="185px">
+        </div>
+      </div>
+      <div class="col-sm-4">
+        <div class="card">
+            <div class="card-block">
+                <h4 class="card-title">MegaPoem ISH II.</h4>
+                
```

Comment:
The format could be improved here. The line spacing doesn't currently indicate different divs or sections. I suggest moving closing tags up (so there's no blank line between), and add blank lines between sections. For this specific section, I'd remove line 89, and add a blank space after 93.